__tests__/app.test.js

```diff
@@ -0,0 +1,64 @@
+const db = require("../db/connection.js");
+const request = require("supertest");
+const app = require("../app");
+const seed = require("../db/seeds/seed");
+const testData = require("../db/data/test-data/index");
+const { response } = require("../app");
+
+beforeEach(() => {
+  return seed(testData);
+});
+afterAll(() => {
+  return db.end();
+});
+
+describe("Get /api/categories", () => {
+  test("Function able to recieve a positive response", () => {
+    return request(app).get("/api/categories").expect(200);
```

Comment:
This assertion has also been indirectly tested for in the below test as well, so you do not need to separate these two out.